packages/client/docusaurus/docs/client/04-call-engine/02-state-store.mdx

```diff
@@ -3,8 +3,84 @@ id: 02-state-store
 title: State Store
 ---
 
-Stay tuned for more information on this topic.
+Our SDK uses a design pattern that we refer to as a reactive state store.
 
-[//]: # 'State store (raw/hooks)'
-[//]: # '- introduction- a central source of truth, core parts working with store, exposed to use by SDKs (React/RN/Angular, or React bindings).'
-[//]: # '- Code examples- what does the integ. gain from code (with hooks/raw)'
+The `StreamVideoClient` instance manages its own state store and each `Call` instance has its own state store as well.
```

Comment:
I propose to mention, what state the client holds compared to the that of Call. It is not clearly distinguished in this article.

Reply:
Do you have a suggestion for this? My idea was that integrators will know what state means by looking at the list of hooks that we reference below

Reply:
What could help here is to formulate what was our aim, why have we done things in a certain way. So we can answer the reader question "Why have we separated client and call states?". Can we say, what is the general purpose of the client state vs that of the call?

I have a feeling that if we force the reader to look at the reference list, we force them to perform this mental exercise similar to reading the source code. And then they have to make some synthesis on their own. Maybe we can provide them some anchor upfront.

Reply:
> Why have we separated client and call states?

The client and the call are two separate entities, so I think it's expected that they have separate states

> what is the general purpose of the client state vs that of the call

I've added examples to show what kind of data are stored in each store